tests/lib/prepare.sh

```diff
@@ -345,6 +345,26 @@ repack_snapd_snap_with_deb_content() {
     rm -rf "$UNPACK_DIR"
 }
 
+repack_core20_snap_with_tweaks() {
+    local CORE20SNAP="$1"
+    local TARGET="$2"
+
+    local UNPACK_DIR="/tmp/core20-unpack"
+    unsquashfs -no-progress -d "$UNPACK_DIR" "$CORE20SNAP"
+
+    mkdir -p "$UNPACK_DIR"/etc/systemd/journald.conf.d
+    cat <<EOF > "$UNPACK_DIR"/etc/systemd/journald.conf.d/to-console.conf
+[Journal]
+ForwardToConsole=yes
+EOF
+    echo 'SNAPD_DEBUG=1' >> "$UNPACK_DIR"/etc/environment
```

Comment:
This makes the smoke test fail now because it does not expect stderr output when running commands, I think this idea is great but maybe we want something more targeted, let me try something